presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/InlineProjections.java

```diff
@@ -172,6 +212,26 @@ private Expression inlineReferences(Expression expression, Assignments assignmen
         return Sets.union(singletons, constants);
     }
 
+    private Set<VariableReferenceExpression> extractTryArguments(RowExpression expression, TypeProvider types)
+    {
+        if (isExpression(expression)) {
+            return extractTryArguments(castToExpression(expression), types);
```

Comment:
inline `extractTryArguments` for `Expression` into this block.

Reply:
Comment https://github.com/prestodb/presto/pull/13047/commits/2a5f744ca40b7b330ac5056b5b317eab35683a2c#r312617399 was not addressed